ods_ci/tests/Resources/Common.robot

```diff
@@ -176,17 +176,31 @@ Get CodeFlare Version
 #robocop: disable: line-too-long
 Wait Until Csv Is Ready
   [Documentation]   Waits ${timeout} for Operators CSV '${display_name}' to have status phase 'Succeeded'
-  [Arguments]    ${display_name}    ${timeout}=10m    ${operators_namespace}=openshift-operators
+  [Arguments]    ${display_name}    ${timeout}=10m    ${operators_namespace}=openshift-operators    ${check_interval}=5s
   Log    Waiting ${timeout} for Operator CSV '${display_name}' in ${operators_namespace} to have status phase 'Succeeded'    console=yes
   WHILE   True    limit=${timeout}
   ...    on_limit_message=${timeout} Timeout exceeded waiting for CSV '${display_name}' to be created
+    Sleep  ${check_interval}
     ${csv_created}=    Run Process    oc get csv --no-headers -n ${operators_namespace} | awk '/${display_name}/ {print \$1}'    shell=yes
-    IF    "${csv_created.stdout}" == "${EMPTY}"    CONTINUE
+    Log To Console    The Result of csv_created Output is: ${csv_created.stdout} RC: ${csv_created.rc}
+    IF  '$csv_created.stdout' == '${EMPTY}'    CONTINUE
```

Comment:
why changing from "${csv_created.stdout}" to '$csv_created.stdout'?  it is a regular string check, the original version should work just fine

Reply:
but this is not a new code, was it broken before?